roles/kubernetes/control_plane/init/primary/templates/kubeadm-config.j2

```diff
@@ -7,14 +7,21 @@
 #
 # kubeadm Configuration : https://kubernetes.io/docs/reference/config-api/kubeadm-config.v1beta2/
 #
-{% if kubernetes_version is version('1.26.0', 'lt', version_type='semver') %}
-apiVersion: kubeadm.k8s.io/v1beta2
-{% else %}
+{% if kubernetes_version is version('1.31.0', 'lt', version_type='semver') %}
```

Comment:
Is this a breaking change? Asking as you've removed the previous logic. If the code is no longer compatible with any previous version, then its a breaking change. This is a process and logistical call on your teams part. Make sure this one creates a new primary version of the tooling.